config/environments/production.rb

```diff
@@ -77,3 +77,13 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 end
+
+ActionMailer::Base.smtp_settings = {
+  user_name: ENV.fetch("SENDGRID_USERNAME"),
+  password: ENV.fetch("SENDGRID_PASSWORD"),
+  domain: "anyratrescue.org",
+  address: 'smtp.sendgrid.net',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.